Docs 5527 update rtm intro (#355)
mulesoft/docs-runtime-manager: modules/ROOT/pages/index.adoc

endif::[]
:keywords: cloudhub, cloud, saas, applications, servers, clusters, sdg, runtime manager, arm

image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
image:logo-rtf-active.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]

Anypoint Runtime Manager is the interface to https://anypoint.mulesoft.com[Anypoint Platform]
that provides a unified view of your applications, servers, and APIs.

With Runtime Manager, you can deploy, manage, and monitor your Mule applications in a sandbox, staging,
or production environment from one central location, whether your apps are deployed in the cloud or on-premises.

You can use Runtime Manager in these scenarios: 

CloudHub::
MuleSoft hosts your app deployments in CloudHub and you use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them.
+

> CloudHub is a complete integration platform as a service (iPaaS) that covers all of your server needs out of the box, including object store infrastructure, monitoring, troubleshooting, and scheduling.

Reviewer: ```suggestion
CloudHub is a complete integration platform as a service (iPaaS) that automatically meets all of your server needs without requiring any configuration or customization, including object store infrastructure, monitoring, troubleshooting, and scheduling.
```

Hybrid::

> You host your app deployments on your on-premises Mule servers and use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them.

Reviewer: Is this correct? parallelism
```suggestion
Your on-premises Mule servers host your app deployments and you use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them.
```

Anypoint Platform Private Cloud Edition::

> You host your app deployments on your local Mule servers and use the on-premises Runtime Manager instance 

Reviewer: ditto re. parallel with first list item:
```suggestion
Your local Mule servers host your app deployments and you use the on-premises Runtime Manager instance 
```

included with Anypoint Platform PCE to manage them.
+
If you are using the on-premises Runtime Manager, you can deploy anywhere _except_ CloudHub.

Anypoint Runtime Fabric::

> You host your app deployments on your locally installed Runtime Fabric instance and use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them. 

Reviewer: ditto; reword like the others

// image::arm-big-picture.png[]
image::rtm-overview.png[]

For applications deployed to CloudHub, you can use Insight for analytics and monitoring.

> For applications that you deploy on-premises or to your private cloud servers, you can send data to third-party analytics applications.

Reviewer: Would there be any advantage to your audience to have the analytics info in each of the above descriptions?

[NOTE]
When you run an app in Anypoint Studio, the app deploys to an embedded test server in Studio.
Because this server is not meant for production deployment and uptime restrictions apply,
deploy your Mule app using one of the deployment options supported by Runtime Manager.

== See Also

> * xref:cloudhub.adoc[CloudHub]

Reviewer: I hope we can expand this title at some point....

* xref:private-cloud::index.adoc[Anypoint Platform Private Cloud Edition]
* xref:runtime-fabric::index.adoc[Anypoint Runtime Fabric]
* xref:deploying-to-cloudhub.adoc[Deploy to CloudHub]
* xref:deploying-to-your-own-servers.adoc[Deploy to Your Servers]
* xref:deployment-strategies.adoc[Deployment Options]

> * xref:insight.adoc[Insight]

Reviewer: ditto
